### PR TITLE
327 - Block 2 - Flip cards

### LINKED ISSUE #327: Block 2  - Flip cards
**User story:**
As a user I would like to be able to add flip cards so that I can display cards with an interactive animation

This block is created to provide a replacement for the ION component seen in [content-at-scale-offering](https://expertise.netcentric.biz/content-at-scale-offering)

**AC:**

- Each card has a title + pre-title, and a text 
- Each card has a hover event
- Pre-title and title appear on both the active(expanded) and default view
- Colour of card background default status is different from the hover effect
- Each card has a different colour
- Hover state card has a  border 1 px 
- Hover state text colour different to default state text colour
- Text allows lists, links and Bold/underline/cursive design
- Pre-title and title are left aligned 
- When the item is in focus the section expands with animation
- Limited number of lines according to design
- Number of cards is always even 
- Min of 2 cards 

**Specs:**
[Block 2- Flip cards with hover effect](https://projects.netcentric.biz/wiki/display/NCWEB/Block+2-+Flip+cards+with+hover+effect?src=contextnavpagetreemode)
Colour guidelines: [Accessibility colour recommendation](https://projects.netcentric.biz/wiki/display/NCWEB/Accessibility+colour+recommendation?src=contextnavpagetreemode)

### PR DESCRIPTION
**Pull request by mihai-buricea-net**
Fix #327 

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/what-we-do/content-at-scale
- After: https://327-flip-cards--cn-website--netcentric.hlx.page/drafts/mburicea/what-we-do/content-at-scale-demo

### CODE REVIEW

File: blocks/flip-cards/flip-cards.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const [defaultState, hoverState] = row.children;
+
+    defaultState.classList.add('flip-cards-default');
+    hoverState.classList.add('flip-cards-hover');
+
+    hoverState.querySelectorAll('li a').forEach((link) => {
+      const chevron = document.createElement('span');
+      chevron.classList.add('icon', 'icon-chevron-right');
+      link.classList.add('button');
+      link.append(chevron);
```
Comment (nc-andreashaller) [inline blocks/flip-cards/flip-cards.js:15]: the links behave a little strange for me (chrome MacOS), they show the link cursor first but after a second switch back to the normal mouse cursor, clicking works, it is just a little irritating

Further I don't now if we should implement a dedicated solution for links in the flip card or rather use the default link behaviour we have everywhere else

**Review by nc-andreashaller — APPROVED** [review]
@tomasznetcentric please review especially CSS as I did not really look in detail there :D

**Review by tomasznetcentric — APPROVED** [review]
well done

File: blocks/flip-cards/flip-cards.js (hunk 1 above)
Comment (mihai-buricea-net) [reply blocks/flip-cards/flip-cards.js:15]: do you mean that hand (pointer) cursor is only visible the first time you hover over the links? I can't reproduce it.

> Further I don't now if we should implement a dedicated solution for links in the flip card or rather use the default link behaviour we have everywhere else
It was decided the link should have the dark plum color and add a chevron. @DaphneKoren has more infor about it.

Comment (nc-andreashaller) [reply blocks/flip-cards/flip-cards.js:15]: The link hover is a bit strange I can show you in a short screenshare, just ping me when you are available

Comment (nc-andreashaller) [reply blocks/flip-cards/flip-cards.js:15]: @DaphneKoren please reconsider the link requirements for this, in my opinion the links are quite hard to spot now as the only hint is the icon, on top I see no reason why link in the cards should look completely different in those cards as in normal text on the page where they are bold and different color

Comment (NCalexiaeche) [reply blocks/flip-cards/flip-cards.js:15]: @catarinajordao What do you suggest here?

**Pull request merged**